src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -93,5 +78,31 @@ def set_connection_string(self, server, port, db_name, username, password):
             Database password.
         """
 
+        assert type(server) is str, "Argument `server` must be a string`"

```

Comment:
It would be nice defining a single place for error messages, then importing it and replacing values where needed. So we can avoid hardcoding repetitive strings in the code.

Reply:
That's true, I am rewriting that same text over and over. That may be worthwhile.